### DISCUSSION TITLE
doris backendid重启后变化

### DISCUSSION THREAD

**Discussion by cst4049** [apache/doris#25454] [A - General / Q&A]
doris k8s部署的2.0.1版本
只有一个be一个fe, 机器上重启be后，backendid 从10002变成了11002, 多次重启后节点ID攀升到17005, 节点也有短暂不可查询， 需要等到它内部backendid 刷新完成后才能支持查询
![image](https://github.com/apache/doris/assets/16145670/13b7dc20-f806-4028-86dd-502f401903c3)

**Comment by cst4049** [comment]
这个是什么原因导致的呢， 重启动同一个be, beid应该是保持不变的吧， 每次重启都有可能导致beid变化， 存储的数据也需要迁移应该会很占用资源吧？应该怎么避免呢有了解的大佬么

**Reply by yagagagaga** [reply]
Which way you use to deploy Doris on K8s?  Yaml or operator?

**Reply by yagagagaga** [reply]
You can refer to this discussion:
https://github.com/apache/doris/discussions/24551
